File: Python/Apuntes/Bases de datos/MySQL/Cliente.py
class Cliente:
    def __init__(self, id=None, nombre=None, apellido=None, membresia=None):
        self.__id = id
        self.__nombre = nombre
        self.__apellido = apellido
        self.__membresia = membresia

    def __str__(self):
        return (f'Id: {self.__id}, Nombre: {self.__nombre}, '
                f'Apellido: {self.__apellido}, Membresia: {self.__membresia}')

    def set_id(self, id):
        self.__id = id

    def set_nombre(self, nombre):
        self.__nombre = nombre

    def set_apellido(self, apellido):
        self.__apellido = apellido

    def set_membresia(self, membresia):
        self.__membresia = membresia

File: Java/ZonaFitSpring/src/main/java/gm/zona_fit/modelo/Cliente.java
package gm.zona_fit.modelo;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.*;

@Entity
@Data //Para generar los metodos get y set automaticamente
@NoArgsConstructor //Agregar un constructor vacio
@AllArgsConstructor //Agregar un constructor con todos los parametos
@ToString //Agregar metodo toString
@EqualsAndHashCode //Agrega los metodos de Equal y HashCode
public class Cliente {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY) //Se recomienda cuando se trabaja con mySql y columnas de valor autoincrementable
    private Integer id;
    private String nombre;
    private String apellido;
    private Integer membresia;
}
